- Fix lib and pkgconfig directories
- Neuter install-exec-hook target

Index: src/pkcs11/Makefile.am
--- src/pkcs11/Makefile.am.orig
+++ src/pkcs11/Makefile.am
@@ -1,5 +1,7 @@
 include $(top_srcdir)/win32/ltrc.inc
 
+override libdir = $(prefix)/lib/pkcs11
+
 MAINTAINERCLEANFILES = $(srcdir)/Makefile.in $(srcdir)/versioninfo-pkcs11.rc $(srcdir)/versioninfo-pkcs11-spy.rc
 EXTRA_DIST = Makefile.mak versioninfo-pkcs11.rc.in versioninfo-pkcs11-spy.rc.in opensc-pkcs11.pc.in opensc-pkcs11.dll.manifest onepin-opensc-pkcs11.dll.manifest
 
@@ -27,10 +29,6 @@ if WIN32
 OPENSC_PKCS11_LIBS += -lshlwapi
 endif
 
-pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = opensc-pkcs11.pc
-DISTCLEANFILES = $(pkgconfig_DATA)
-
 opensc_pkcs11_la_SOURCES = $(OPENSC_PKCS11_SRC) $(OPENSC_PKCS11_INC)
 opensc_pkcs11_la_CFLAGS = $(OPENSC_PKCS11_CFLAGS)
 opensc_pkcs11_la_LIBADD = $(OPENSC_PKCS11_LIBS)
@@ -81,6 +79,7 @@ uninstall-hook:
 else
 # see http://wiki.cacert.org/wiki/Pkcs11TaskForce
 install-exec-hook:
+we-dont-do-that-here:
 	$(MKDIR_P) "$(DESTDIR)$(pkcs11dir)"
 	for l in opensc-pkcs11$(DYN_LIB_EXT) onepin-opensc-pkcs11$(DYN_LIB_EXT) pkcs11-spy$(DYN_LIB_EXT); do \
 		rm -f "$(DESTDIR)$(pkcs11dir)/$$l"; \
